app/src/androidTest/java/com/stevesoltys/seedvault/e2e/screen/impl/BackupScreen.kt

```diff
@@ -14,5 +14,11 @@ object BackupScreen : UiDeviceScreen<BackupScreen>() {
 
     val backupLocationButton = findObject { text("Backup location") }
 
+    val backupSwitch = findObject { text("Backup my apps") }
+
+    val internalStorageButton = findObject { textContains("Android SDK built for") }
```

Comment:
maybe nice to use the viewID or something more generic here, so it will work on different devices as well.

Reply:
Agreed, but for some reason I couldn't get any of elements selected by resource id.

Struggled with it for a long time before giving up for now.

Reply:
i.e.:

```
   val internalStorageButton = findObject {
        resourceIdMatches(".*titleView.*")
    }
```

or any combination of this just ... doesn't work :disappointed: